README.md

```diff
@@ -38,6 +38,25 @@ All updates to this library is documented in our [CHANGELOG](https://github.com/
 ```bash
 go get github.com/sendgrid/smtpapi-go
 ```
+
+## Setup Environment Variables
+
+### Initial Setup
+
+```bash
+cp .env_sample .env
```

Comment:
If you are doing this step, then all you need to do is open and edit the file by inserting your API KEY.

The next section `environment variable` is redundant.